tests/xcconfig/BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+load("//rules:framework.bzl", "apple_framework")
+load(":tests.bzl", "xcconfig_unit_test_suite")
+
+SRCS = [
+    "empty.m",
+    "empty.swift",
+]
+
+EXPECTED_FAILURE = ["manual"]
+
+apple_framework(
+    name = "empty_xcconfig",
+    srcs = SRCS,
+    xcconfig = {},
+)
+
+apple_framework(
```

Comment:
Are you just invoking these rule implementations from these tests to ensure that the rule implementation doesn't break when these different xcconfig values are passed in?

Reply:
Correct